src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -94,10 +94,6 @@
  *  using jcabi-http, provide a maven configuration file with proxy settings
  *  and verify the expected HTTP requests have been sent to the mocked server.
  * @todo #69:30min Fix UnsupportedClassVersionError in CL build for Open JDK6.
- * @todo #82:30min Clean unnecessary docker setup (src/docler/Dockerfile),
- *  make the configuration minimal. Set versions to the used packages
- *  (e.g. lock java version instead of default jdk). After changes, the file
- *  needs to be uploaded to Dockerhub and be built there.
```

Comment:
@mkordas actually putting the file to that repo only was the solution for commercial repos, because we didn't want to expose the whole repo publicly in that case.
For open source stuff, it should also be fine here, but i @yegor256 want's to keep things in one place that's his choice, works too :)